.cloudbuild/ci/os-compatibility-test.yaml

```
@@ -0,0 +1,23 @@
+timeout: 25m
+
+options:
+   machineType: E2_HIGHCPU_32
+
+steps:
+  - name: public.ecr.aws/gravitational/teleport-buildbox-centos7:teleport11
+    id: build-teleport
+    dir: /workspace
+    entrypoint: "/bin/bash"
+    args: 
+      - '-c'
+      - 'make -j$$(nproc) build/tctl build/tsh build/tbot build/teleport'  
```

```suggestion
      - 'make -j$$(nproc) all'  
```

Won't this have the same effect? If so, it's less typing and we don't have to update it if we add a binary.